src/components/navigation-bars/mobile-navigation-bar.tsx

```diff
@@ -1,176 +1,91 @@
 import { useRouter } from "next/router";
 import React from "react";
-import styled from "styled-components";
 
-import { ClickAwayListener } from "../click-away-listener";
 import { Arrow } from "../icons/arrow";
 import { BurgerIcon } from "../icons/burger-icon";
 import { NavBarCrossIcon } from "../icons/navbar-cross-icon";
 import { RightChevron } from "../icons/right-chevron";
-import { WhiteSwitchIcon } from "../icons/switch-icon/white-switch-icon";
 import { WhiteWorldIcon } from "../icons/world-icon/white-world-icon";
 import { LogoutAnchor } from "../logout-anchor";
+import { ModalOverlay } from "../modal-overlay";
 import { NeutralLink } from "../neutral-link";
 import { getNavigationSections } from "./navigation-sections";
 import { formatNavigation } from "@src/configs/intl";
-import { deviceBreakpoints } from "@src/design-system/theme";
 
 const MobileNavigationBar: React.FC = () => {
   const [isOpen, setIsOpen] = React.useState<boolean>(false);
   const router = useRouter();
 
   return (
     <>
-      {isOpen ? <ClickAwayListener onClick={() => setIsOpen(false)} /> : null}
-      <Container>
+      {isOpen ? <ModalOverlay onClick={() => setIsOpen(false)} /> : null}
+      <div className="block lg:hidden fixed bottom-0 left-0 w-full bg-background z-20">
```

Comment:
Could you give more precision in the naming of `isOpen`? Like `isModalOpen`?

Reply:
Sure 🙇‍♀️